pkg/apis/pipeline/v1beta1/pipelinerun_conversion.go

```diff
@@ -235,8 +238,7 @@ func (ptrs *PipelineTaskRunSpec) convertFrom(ctx context.Context, source v1.Pipe
 	ptrs.ComputeResources = source.ComputeResources
 }
 
-// ConvertTo implements apis.Convertible
-func (prs *PipelineRunStatus) ConvertTo(ctx context.Context, sink *v1.PipelineRunStatus) error {
+func (prs *PipelineRunStatus) convertTo(ctx context.Context, sink *v1.PipelineRunStatus) error {
```

Comment:
just want to doublecheck there are no issues to make these functions private?

Reply:
Thanks Yongxuan and yeah I think these changes are good:) We are not importing on PipelineRunStatus Conversion elsewhere.

Reply:
`PipelineRunStatus` doesn't need to implement apis.Convertible, and we don't rely on this anywhere else in pipelines (and probably not other repos but I haven't checked). We don't have compat policies defined for our go libraries, although this technically isn't a backwards compatible change. I can add a release note for anyone who might be using this method.